modules/ROOT/pages/_partials/flex-tls-cipher.adoc

```diff
@@ -30,6 +30,21 @@ IMPORTANT: Ensure that your API upstream supports the selected ciphers and versi
 
 //end::cipherSupport[]
 
+//tag::cipherSupportLocal[]
+[[local-support]]
+== TLS Cipher Support on Flex Gateway
+
+Flex can support a range of TLS Versions from TLS 1.1 to TLS 1.3, and you can also customize some of the ciphers to support.
+
+You cannot customize the list of TLS 1.3 Ciphers. If you support TLS 1.3, the TLS 1.3 default ciphers are all supported regardless of the ciphers listend in the configuration file. If you don't support TLS 1.3, these ciphers are not included.
+
+For TLS 1.2, listing any ciphers overides the TLS 1.2 default ciphers. If you want to list ciphers besides the default TLS 1.2 Ciphers, you must list every cipher to support including the default ciphers you want to support. Excluding default ciphers from your list of supported ciphers means you do not support those ciphers. If you wish to only support the default ciphers, you can leave the cipher list blank. Listing ciphers does not affect the default TLS 1.3 cipher.  
+
+There are no default ciphers for TLS 1.1. To support TLS 1.1, you must list the ciphers to support. If you support TLS 1.1 and TLS 1.2, listing TLS 1.1 ciphers overides the TLS 1.2 defaults. To support both these versions, list all ciphers you want to support.
+
+IMPORTANT: Ensure that your API upstream supports the selected ciphers and versions.
```

Comment:
This is only required when TLS context is used for outbound. 
For inbound, since Flex is terminating the TLS connection, there is no need for the upstream API to support the ciphers. The connection to the upstream may even be HTTP.